include/dev/MAX31855.h

```diff
@@ -0,0 +1,39 @@
+#ifndef TMU_MAX31856_H
```

Comment:
Rename this file to `MAX31855.hpp` to match the rest of our header files. Generally, we use `.h` with `.c` files, and `.hpp` with `.cpp` files, although `.h` works for both `.c` and `.cpp` files.